course/auth.py

```diff
@@ -209,12 +228,17 @@ def __init__(self, *args, **kwargs):
         self.helper.add_input(Submit("submit", _("Impersonate")))
 
 
-def impersonate(request):
+def impersonate(request, course_identifier):
```

Comment:
I wonder if we should have a non-course entry point (URL) for the (global) impersonation. (Not for me, but I'm thinking of other folks serving as admins of Relate instances.)

Reply:
Would they be superusers? Or users with a relate permission level?

Reply:
> Would they be superusers?

Yep.